### PR TITLE
test(cli): add test cases

### LINKED ISSUE #16090: Not enough test case in cli/display.rs
Since `size < 1_f64 ` determines whether the value is negative or not, `let negative = ...` does not seem to be necessary.

https://github.com/denoland/deno/blob/927f4e2e83719aac3dcc4d9ae422cbbf76bd7bcd/cli/display.rs#L6-L11

### PR DESCRIPTION
**Pull request by skanehira**
fix #16090

### CODE REVIEW

**Comment by cjihrig** [comment]
Unless I'm misunderstanding something, I don't think this change is correct (https://github.com/denoland/deno/issues/16090#issuecomment-1262514377).

File: cli/display.rs [hunk 1]
```diff
@@ -3,11 +3,10 @@
 /// A function that converts a float to a string the represents a human
 /// readable version of that number.
 pub fn human_size(size: f64) -> String {
-  let negative = if size.is_sign_positive() { "" } else { "-" };
   let size = size.abs();
   let units = ["B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
   if size < 1_f64 {
-    return format!("{}{}{}", negative, size, "B");
+    return format!("-{}{}", size, "B");
```
Comment (dsherret) [inline cli/display.rs:9]: I think this would add a negative sign for `human_size(0_f64)`?

Comment (skanehira) [reply cli/display.rs:9]: Oh sorry, my understanding was wrong 😅
this change is not correct.
But test cases seems like not enough (e.g -10GB)
I’ll revert and add test cases in this PR.

Comment (skanehira) [reply cli/display.rs]: Done revert and add tests :)

**Review by cjihrig — APPROVED** [review]

**Pull request merged**